pkg/scheduler/objects/application.go

```diff
@@ -312,9 +330,14 @@ func (sa *Application) timeoutPlaceholderProcessing() {
 		log.Logger().Info("Placeholder timeout, releasing asks and placeholders",
 			zap.String("AppID", sa.ApplicationID),
 			zap.Int("releasing placeholders", len(sa.allocations)),
-			zap.Int("releasing asks", len(sa.requests)))
+			zap.Int("releasing asks", len(sa.requests)),
+			zap.String("used Scheduling type", sa.gangSchedulingStyle))
```

Comment:
suggest to change "used scheduling type" to "gang scheduling style", just to keep the words consistent